sql/original_update_tests.md

```diff
@@ -0,0 +1,48 @@
+# Original Update Tests
+This file is a port of the original hardcoded update-tester tests to markdown, similar to how sql-doctester works.
+
+These markdown files are parsed in the same way as sql-doctester would with the addition of two fields: `creation` and `validation`. (Names are not final).
+
+Tests marked `creation` will be ran during the object creation step, tests marked validation will be ran during the validation step.
```

Comment:
These two lines are examples of what I mean! :)

It may be convenient for you to place these comments here, but:
- They could as easily have been SQL comments
- The long-term intent is unclear.